airbyte-integrations/connectors/source-facebook-marketing-api-singer/requirements.txt

```diff
@@ -0,0 +1,5 @@
+-e ../../bases/airbyte-protocol
+-e ../../bases/base-singer
+-e ../../bases/base-python
```

Comment:
why does it need base-python?

Reply:
we use some classes from base-python for type annotations